dev_guide/quota.adoc

```diff
@@ -87,116 +126,74 @@ in the system.
+for either CPU or memory.  The quota can be configured to quota either value.  If the quota
+has a usage limit specified for requests.cpu or requests.memory, then it will require that 
+every incoming container makes an explicit request for those resources.  If the quota has a 
+usage limit specified for limits.cpu or limits.memory, then it will require that every incoming
+container specifies an explicit limit for those resources.
 
-|pod-y
-|b
-|100m
-|none
-|The quota usage is incremented 100m
+== Sample Resource Quota Files
 
-|pod-y
-|c
-|none
-|500m
-|The quota usage is incremented 500m since request will default to limit.
-
-|pod-z
-|d
-|none
-|none
-|The pod is rejected since it does not enumerate a request.
-
-|===
-
-The rationale for charging quota usage for the requested amount of a resource versus the limit is the belief that a user should only be charged for what they are scheduled against in the cluster.
-
-As a consequence, the user is able to spread its usage of a resource across multiple tiers of service. Let's demonstrate this via an example with a 4 cpu quota.
-
-The quota may be allocated as follows:
-
-.Quota with Burstable resources
-[cols="3a,3a,3a,3a,3a,8a",options="header"]
-|===
-
-|Pod |Container |Requests[CPU] |Limits[CPU] |Quality of Service Tier |Quota Usage
-
-|pod-x
-|a
-|1
-|4
-|Burstable
-|1
-
-|pod-y
-|b
-|2
-|2
-|Guaranteed
-|2
-
-|pod-z
-|c
-|1
-|3
-|Burstable
-|1
-
-|===
-
-The user is restricted from using *BestEffort* CPU containers, but at any point in time, the containers on the node may consume up to 9 CPU cores over a given time period if there is no contention on the node because it
-has **Burstable** containers.  If one wants to restrict the ratio between the request and limit, it is encouraged that the user define a `*LimitRange*` with `*MaxLimitRequestRatio*` to control burst out behavior. This would in effect, let an administrator keep the difference between request and limit more in line with tracked usage if desired.
+object-counts.yaml
+====
+----
+apiVersion: v1
+kind: ResourceQuota
+metadata:
+  name: object-counts
+spec:
+  hard:
+    configmaps: "10" <1>
+    persistentvolumeclaims: "4" <2>
+    replicationcontrollers: "20" <3>
+    secrets: "10" <4>
+    services: "10" <5>
+----
+<1> The total number of config maps in the project
+<2> The total number of persistent volume claims in the project
+<3> The total number of replication controllers in the project
+<4> The total number of secrets in the project
+<5> The total number of services in the project
+====
 
-== Sample Resource Quota File
+compute-resources.yaml
+====
+----
+apiVersion: v1
+kind: ResourceQuota
+metadata:
+  name: compute-resources
+spec:
+  hard:
+    pods: "4" <1>
+    requests.cpu: "1" <2>
+    requests.memory: 1Gi <3>
+    limits.cpu: "2" <4>
+    limits.memory: 2Gi <5>
+----
+<1> The total number of pods that are in a non-terminal state in the project
+<2> The total number of cpu requests in the project
```

Comment:
Does this mean that across all pods in this project, the sum of the cpu requests must be <= 1 cpu?